microsoft/testsuites/vm_extensions/cvt.py

```diff
@@ -0,0 +1,284 @@
+        "SLES11-SP4-64",
+        "OL6-64",
+        "RHEL7-64",
+    }
+    publisher_name = "Microsoft.Azure.SiteRecovery.Test"
+    extension_name = _get_extension_name(os=os, log=log)
+    if os in extension_publishers:
+        if os in extension_test:
+            extension_name = extension_name + "Test"
+    else:
+        publisher_name = "Microsoft.Azure.SiteRecovery2.Test"
+
+    cur_time = datetime.datetime.now().isoformat() + "Z"
+    settings = {
+        "publicObject": "",
+        "module": "a2a",
+        "timeStamp": cur_time,
+        "commandToExecute": "Install",
+        "taskId": task_id,
+    }
+    extension = node.features[AzureExtension]
+
+    result = extension.create_or_update(
+        name=publisher_name + "." + extension_name,
+        publisher=publisher_name,
+        type_=extension_name,
+        type_handler_version="1.0",
+        auto_upgrade_minor_version=True,
+        settings=settings,
+    )
+    assert_that(result["provisioning_state"]).described_as(
+        "Expected the extension to succeed"
+    ).is_equal_to("Succeeded")
+
+
+def _run_script(
+    node: Node, log: Logger, test_dir: str, cvt_script: CustomScriptBuilder
+) -> ExecutableResult:
+    timer = create_timer()
+    script: CustomScript = node.tools[cvt_script]
+    result = script.run(parameters=test_dir, timeout=19800, sudo=True)
+    log.info(f"Script run with param {test_dir} finished within {timer}")
+    return result
+
+
+def _copy_cvt_logs(
+    log: Logger,
+    log_path: Path,
+    node: Node,
+    test_dir: Path,
+) -> None:
+    find_tool = node.tools[Find]
+    file_list = find_tool.find_files(
+        test_dir,
+        name_pattern="*.log",
+        file_type="f",
+        sudo=True,
+        ignore_not_exist=True,
+    )
+
+    file_list += find_tool.find_files(
+        test_dir,
+        name_pattern="*.txt",
+        file_type="f",
+        sudo=True,
+        ignore_not_exist=True,
+    )
+
+    for file in file_list:
+        log.info(f"Copying file {file} to {log_path}")
+        try:
+            file_name = file.split("/")[-1]
+            node.shell.copy_back(
+                node.get_pure_path(file),
+                log_path / f"{file_name}",
+            )
+        except FileNotFoundError:
+            log.error(f"File {file} doesn't exist.")
+
+
+def _run_cvt_tests(
+    log: Logger,
+    node: Node,
+    log_path: Path,
+    variables: Dict[str, Any],
+    os: str,
+    cvt_script: CustomScriptBuilder,
+) -> Optional[int]:
+    cvt_bin = "indskflt_ct"
+    container_sas_uri = variables.get("cvtbinaries_sasuri", "")
+    if not container_sas_uri:
+        log.error("sas uri for cvt binary is empty.")
+        raise SkippedException("cvt binary is not provided.")
+    cvt_binary_sas_uri = container_sas_uri.replace(
+        "?", "/cvtbinaries/indskflt_ct_" + os + "?"
+    )
+
+    cvt_root_dir = str(node.working_path) + "/LisaTest/"
+    cvt_download_dir = cvt_root_dir + "cvt_files/"
```

Comment:
It doesn't need the cvt_root_dir


```suggestion
    cvt_download_dir = str(node.working_path / "cvt_files")
```